ipl/main.c

```diff
@@ -665,31 +665,52 @@ void print_tsec_key()
 	{
 		EPRINTFARGS("Could not identify package1 version\nto read TSEC firmware (= '%s').",
 			(char *)pkg1 + 0x10);
+		
+		gfx_puts(&gfx_con, "\nPress any key...\n");
+		btn_wait();
```

Comment:
Remove these 2.
Press any key is not needed anymore.
btn_wait is going to be used elsewhere.